change-control-lambda/lib/time-window.ts

```diff
@@ -40,10 +42,66 @@ export function shouldBlockPipeline(icalData: string | Buffer, now = new Date(),
   return blocks.length > 0 ? blocks[0] : undefined;
 }
 
+/**
+ * Returns the previous and next events for a recurring event surrounding the
+ * provided date. If the date provided is equal to the start of an event, 
+ * the event for that date and the following event will be returend
+ * 
+ * If the provided event is not recurring, no events are returned.
+ *
+ * @param event a recurring calendar event.
+ * @param date the date for which the previous and next event should be returned.
+ */
+function getRecurringEvents(event: CalendarEvent, date: Date): CalendarEvent[] {
+  const recurrences: CalendarEvent[] = [];
+  if (event.rrule == null) return recurrences;
+  const duration = (new Date(event.end).getTime() - new Date(event.start).getTime())
+
+  const after = event.rrule.after(date, false);
```

Comment:
This tells me that the type of `rrule` is not `any`. What's the correct type that should be assigned here?

Reply:
I'll pull in the type from RRule and add it here.